frontend/src/k8sTypes.ts

```diff
@@ -344,11 +344,17 @@ export type PodKind = K8sResourceCommon & {
   status?: {
     phase: string;
     conditions: K8sCondition[];
-    containerStatuses?: {
-      name?: string;
-      ready: boolean;
-      state?: { running?: boolean; waiting?: boolean; terminated?: boolean };
-    }[];
+    containerStatuses?: PodContainerStatus[];
+  };
+};
+
+export type PodContainerStatus = {
+  name: string;
```

Comment:
Moved the PodContainerStatus from `frontend/src/types.ts` to `frontend/src/k8sTypes.ts` and verified with @DaoDaoNoCode that `name`   should be a required field